Detectors/TRD/simulation/include/TRDSimulation/TRsim.h

```diff
@@ -0,0 +1,134 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/* Copyright(c) 1998-1999, ALICE Experiment at CERN, All rights reserved. *
+ * See cxx source for full Copyright notice                               */
+
+/* $Id$ */
+
+////////////////////////////////////////////////////////////////////////////
+//                                                                        //
+//  TRD simulation - multimodule (regular rad.)                           //
+//                                                                        //
+////////////////////////////////////////////////////////////////////////////
+#ifndef ALICEO2_TRD_TRSIM_H_
+#define ALICEO2_TRD_TRSIM_H_
+
+#include <TObject.h>
+#include <TMath.h>
+
+class TH1D;
+
+class TRsim : public TObject
```

Comment:
Is the inheritance from TObject necessary?

Reply:
most likely not; please remove

Reply:
You are right, removing.